### ISSUE TITLE
notice on public domain scans

### ISSUE THREAD

**Issue by eroux** [buda-base/public-digital-library#668]
It would be good to have a short sentence on the scans tab for open access public domain works (in the same way that we have one for copyrighted or restricted material), it could be

> The term “public domain” refers to creative materials that are not protected by intellectual property laws such as copyright, trademark, or patent laws. Anyone can use a public domain work without obtaining permission, but no one can ever own it.

@JannTibetan would that be suitable?

**Comment by JannTibetan** [comment]
Good idea. Let me rephrase it so it's more about the practical implications of public domain materials.
